### ISSUE TITLE
Lotto image update

### ISSUE THREAD

**Issue by cosme12** [delight-im/OpenSoccer#272]
I think we should upgrade the lotto drawing. Right now it displays "Ballmanager" so we have no choice but update it.
![lotto](https://cloud.githubusercontent.com/assets/6611118/7552558/ab26584a-f690-11e4-8e55-9886268636b9.png)

We could take advantage of it and make it look less simple, with an improved font or maybe a complete new image. What do you think?

**Comment by mwaclawek** [comment]
I was thinking about removing the lottery altogether? What do you think?

Another solution would be to create some lottery sheet with a `div` for the box, some `span` elements for the numbers and the caption, and the `input type="checkbox"` elements again as they are.

Otherwise, we'd have to create one graphic per language, which isn't that elegant because translating graphics is annoying :)

**Comment by cosme12** [comment]
I think we should keep it. It adds something different to the game, not just full soccer. Also, its a good way to drag money from rich people and re-distribute it.

**Comment by mwaclawek** [comment]
Yes, that's what is was for, but I'm not really sure if that's still working. Anyway, I don't really like the whole page (design, functionality, etc.). But, of course, we can keep it, at least for now :)

I would say we just remove the left part and the top part where the text is. Then again, we could just create 5 rows of 3 checkboxes each with simple HTML.

In addition to that, we currently have no check (before submitting) that ensures people check exactly 4 boxes.
